upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -211,6 +213,16 @@ func (e *InstanceTemplate) Find(c *fi.Context) (*InstanceTemplate, error) {
 		// System fields
 		actual.Lifecycle = e.Lifecycle
 
+		if p.GuestAccelerators != nil {
```

Comment:
```suggestion
```

Reply:
I think, we must have this check. Otherwise `actual.GuestAccelerators` will be an empty slice instead of nil. It currupt back copatibility.

Reply:
Don't think I understand how it can corrupt backward compatibility.

Reply:
All InstanceTemplates created before this implementation will have a difference on applying this implementation:
```
- 
+ guestAccelerators: [],
```

Reply:
This is the kOps representation, not the internal one of GCP. `Find()` will return `[]AcceleratorConfig{}` wether `p.GuestAccelerators` is `nil` or empty slice.

Reply:
fixed